src/reformat_libyuv.c

```diff
@@ -709,11 +889,12 @@ static avifResult avifImageDownshiftTo8bpc(const avifImage * image, avifImage *
     return AVIF_RESULT_OK;
 }
 
-avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
+avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb, avifBool reformatAlpha, avifBool * alphaReformattedWithLibYUV)
 {
     if (rgb->depth != 8 || (image->depth != 8 && image->depth != 10 && image->depth != 12)) {
         return AVIF_RESULT_NOT_IMPLEMENTED;
     }
+    *alphaReformattedWithLibYUV = AVIF_FALSE;
```

Comment:
Nit: Move this to the beginning of the function, so that this function always sets this output parameter.